x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_details/agent_details_overview.tsx

```diff
@@ -142,7 +144,15 @@ export const AgentDetailsOverviewSection: React.FunctionComponent<{
                 defaultMessage: 'Last activity',
               }),
               description: agent.last_checkin ? (
-                <FormattedRelative value={new Date(agent.last_checkin)} />
+                <EuiCopy textToCopy={agent.last_checkin}>
```

Comment:
I am not sure we need a copy button here, it seems having a displayed text value will already allow the user to copy the text value, wdyt?

Reply:
You're right, I think I misinterpreted the copy button ask from the feature request. Seems the Callout box in the integrations input status is not copyable, and the place where that functionality is most needed.

Reply:
Now using formatted date time in https://github.com/elastic/kibana/pull/215531/commits/7993f889d8f140dce1af6691ea50dda00e238302